scripts/call_contracts.py

```diff
@@ -27,11 +27,18 @@ def new_func(ctx: Any, *args: Any, **kwargs: Any) -> Callable:
 
 @click.group("cli")
 @click.option(
-    "--deployment-dir",
+    "--artifact-dir",
     type=Path,
     required=True,
     metavar="DIR",
-    help="The directory that stores contract deployment files.",
+    help="The directory that stores deployment artifact files.",
+)
+@click.option(
+    "--abi-dir",
+    type=Path,
+    required=True,
+    metavar="DIR",
+    help="The directory that stores contract abi files.",
```

Comment:
Nitpick: the `read` command has
```
Path to the directory with contract ABIs.
```
as the help text and we should be consistent.